data/io.elementary.settings-daemon.gschema.xml

```diff
@@ -72,6 +72,11 @@
   </schema>
 
   <schema path="/io/elementary/settings-daemon/system-update/" id="io.elementary.settings-daemon.system-update">
+    <key type="b" name="automatic-updates">
```

Comment:
Should we make this tri-state ahead of time since #52 requested a mode to just install security updates? Is that feasible or should we just say system updates are system updates and you don't get to cherry pick?

Reply:
I would go with system updates are system updates but that's just my opinion 🤷